### PR TITLE
Gunnar: fixed rust dependencies

### PR DESCRIPTION
**Pull request by Gunnar-Stunnar**
Fixed rust dependencies, and pointed them at the Zingolab's repo

### CODE REVIEW

File: native/Cargo.toml [hunk 1]
```diff
@@ -15,6 +15,6 @@ default-features = false
 features = ["napi-6"]
 
 [dependencies]
-zecwalletlitelib = { git = "https://github.com/adityapk00/zecwallet-light-cli", rev = "89be353b5ea600ee21d998254598fb19d8778417" }
+zecwalletlitelib = { git = "https://github.com/zingolabs/zecwalletlitelib", rev = "89be353b5ea600ee21d998254598fb19d8778417" }
```
Comment (zancas) [inline native/Cargo.toml:18]: Shall we change `rev = "...` to `branch = "dev"`..  for `dev`?

**Review by zancas — APPROVED** [review]
Nice!  Thanks!

**Comment by dannasessha** [comment]
not to be too annoying (this just came up today) but it looks like one of these commits wasn't signed which might block a merge :(

**Comment by dannasessha** [comment]
I'm also seeing a package conflict locally... 
```
error: failed to select a version for `http-body`
```

**Comment by dannasessha** [comment]
I was able to supress this error by deleting the `Cargo.lock` file that was also checked in here.

**Comment by dannasessha** [comment]
Other than these issues looks good!

**Comment by zancas** [comment]
Superceded by #24

**Pull request closed without merging**